smoke-test/README.md

```diff
@@ -68,8 +68,8 @@ mvn -DskipITs=false integration-test -Djava.io.tmpdir=/tmp
 ### Run tests from local tarball
 
 If you have the code compiled and assembled locally, you can use the tarball build for container images, so you don't have to wait for the CI/CD to download the container image artifact.
-Drop the assembled OpenNMS-tar.gz file in `opennms-container/horizon/tarball` and run `docker build -t horizon .`
-Smoke tests will run the image named `horizon` in your local Docker image repo.
+Drop the assembled OpenNMS-tar.gz file in `opennms-container/horizon/tarball` and run `docker build -t opennms/horizon .`
+Smoke tests will run the image named `opennms/horizon` in your local Docker image repo.
```

Comment:
Is this even true anymore? Shouldn't we just tell them to run `make image`?

Reply:
For that matter, we could get rid of them altogether, and just have a top-level `Makefile` that just runs `make oci` in each subdir. :)

Reply:
Done, but I didn't add this part, since I didn't see anything that used it (nor mentioned in any of the in-repo docs):
> have a top-level `Makefile` that just runs `make oci` in each subdir. :)

If this is used manually, I can add it in.